aiven/resource_flink_table.go

```diff
@@ -8,6 +8,30 @@ import (
 	"github.com/aiven/aiven-go-client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+const (
+	connectorTypeKafka       = "kafka"
+	connectorTypeUpsertKafka = "upsert-kafka"
+
+	formatAvro                  = "avro"
+	formatAvroConfluent         = "avro-confluent"
+	formatDebeziumAvroConfluent = "debezium-avro-confluent"
+	formatDebeziumJson          = "debezium-json"
+	formatJson                  = "json"
+
+	startupModeEarliestOffset = "earliest-offset"
+	startupModeLargestOffset  = "largest-offset"
+	startupModeGroupOffsets   = "group-offsets"
+	startupModeTimestamp      = "timestamp"
+)
+
+var (
+	aivenFlinkTableConnectorTypes    = []string{connectorTypeKafka, connectorTypeUpsertKafka}
```

Comment:
The problem with the package level variables is that someone can accidentally overwrite it's value. Can use use functions instead? 

Something like:
```golang
func getFlinkTableConnectorTypes() []string {
    return []string{connectorTypeKafka, connectorTypeUpsertKafka}
}
```

Reply:
:heavy_check_mark: